lightstep-tracer-jre-bundle/pom.xml

```diff
@@ -114,6 +132,29 @@
             </execution>
           </executions>
         </plugin>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-shade-plugin</artifactId>
+          <version>3.2.4</version>
+          <executions>
+            <execution>
+              <phase>package</phase>
+              <goals>
+                <goal>shade</goal>
+              </goals>
+              <configuration>
+                <createSourcesJar>true</createSourcesJar>
+                <shadeSourcesContent>true</shadeSourcesContent>
+                <relocations>
+                  <relocation>
+                    <pattern>okhttp</pattern>
```

Comment:
shading only okhttp. 
I tried to shade `io.grpc` and `io.netty` but that doesn't work. Spans are not sent or failure during tracer creation. So for shading of `io.grpc` and `io.netty` we most like need separate issue because it will not be trivial.